### DISCUSSION TITLE
Location of navigation bar and code of the basic pages.

### DISCUSSION THREAD

**Discussion by RohanNayaks** [oqtane/oqtane.framework#1366] [General] [answered]
I am unable to find the location of the navigation bar code and basic pages code.
Please help me out.

**Comment by leigh-pointer** [comment, marked as answer]
Hi @RohanNayaks 
you will find the navigation code in ..\oqtane.framework\Oqtane.Client\Themes\Controls\Theme\ folder

if you are happy with this reply please mark as "Answered"
